### ISSUE TITLE
load_mnist returned buffer

### ISSUE THREAD

**Issue by davidlaiymani** [ml-explore/mlx-data#15]
Great job, thank you. I would like to train a Bert classifier, but how to create a buffer with several "types" of samples like input_ids, attention_masks and labels?  Something similar to the return of load_mnist for instance. In advance thank you.
